specification/app/Microsoft.App/models/common.tsp

```diff
@@ -69,7 +72,7 @@ model SessionPoolNameParameter {
 }
 
 @doc("The code execution name path parameter.")
-model CodeExecutionFileNameParameter {
+model SessionResourceFileNameParameter {
```

Comment:
fileName should be duplicated. 
<img width="398" alt="image" src="https://github.com/user-attachments/assets/6410126b-1117-4013-924d-1a7a8ad5e69b">